pkg/policy/repository.go

```diff
@@ -200,11 +201,12 @@ func NewStoppedPolicyRepository(
 ) *Repository {
 	selectorCache := NewSelectorCache(initialIDs)
 	repo := &Repository{
-		rules:           make(map[ruleKey]*rule),
-		rulesByResource: make(map[ipcachetypes.ResourceID]map[ruleKey]*rule),
-		selectorCache:   selectorCache,
-		certManager:     certManager,
-		secretManager:   secretManager,
+		rules:            make(map[ruleKey]*rule),
+		rulesByNamespace: make(map[string]map[ruleKey]struct{}),
```

Comment:
suggestions: doubly nested maps can make the code harder to read.  It might be cleaner to use `map[string]sets.Set[ruleKey]`

Reply:
Makes sense, I will change it tomorrow

Reply:
Done, I've re-requested review from you :innocent: